recipe/meta.yaml

```diff
@@ -27,11 +27,13 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - maturin 0.14.17
+    - maturin 1.3.1
   run:
     - python
 
 test:
+  source_files:
+    - tests
```

Comment:
you still need to run pytest

Reply:
there are examples around the distro, could search in aggregate for "- pytest" to see what comes up